src/xo-app/backup/new/index.js

```diff
@@ -307,22 +316,25 @@ export default class New extends Component {
         <Section icon='preview' title='preview' summary>
           <div className='card-block'>
             <SchedulePreview cron={this.state.cronPattern} />
-            <fieldset className='pull-xs-right p-t-1'>
-              <ActionButton
-                btnStyle='primary'
-                className='btn-lg m-r-1'
-                disabled={!backupInfo}
-                form='form-new-vm-backup'
-                handler={this._handleSubmit}
-                icon='save'
-                redirectOnSuccess='/backup/overview'
-              >
-                {_('saveBackupJob')}
-              </ActionButton>
-              <button type='button' className='btn btn-lg btn-secondary' onClick={this._handleReset}>
-                Reset
-              </button>
-            </fieldset>
+            {process.env.XOA_PLAN < 4 && backupInfo && DISABLED_BACKUP_TYPES[backupInfo.jobKey] > process.env.XOA_PLAN
```

Comment:
Inverse the direction of the last test as it's more common to have the thing you test for (here the XOA plan) on the left.